winit 0.27 upgrade breaking changes

```rs
error[E0277]: the trait bound `winit::window::Window: HasRawWindowHandle` is not satisfied
   --> crates\head\src\main.rs:350:27
    |
350 |     let surface_texture = SurfaceTexture::new(window_size.width, window_size.height, &window);
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `HasRawWindowHandle` is not implemented for `winit::window::Window`
    |
    = help: the trait `HasRawWindowHandle` is implemented for `&'a T`

error[E0277]: the trait bound `winit::window::Window: HasRawWindowHandle` is not satisfied
   --> crates\head\src\main.rs:351:71
    |
351 |     let pixels = if let Ok(pixels) = Pixels::new_async(WIDTH, HEIGHT, surface_texture).await {
    |                                      -----------------                ^^^^^^^^^^^^^^^ the trait `HasRawWindowHandle` is not implemented for `winit::window::Window`
    |                                      |
    |                                      required by a bound introduced by this call
    |
    = help: the trait `HasRawWindowHandle` is implemented for `&'a T`

note: required by a bound in `pixels::Pixels::new_async`
   --> C:\Users\Zageron\.cargo\git\checkouts\pixels-fd3805be475df9df\0b15a5d\src\lib.rs:227:31
    |
227 |     pub async fn new_async<W: HasRawWindowHandle>(
    |                               ^^^^^^^^^^^^^^^^^^ required by this bound in `pixels::Pixels::new_async`
```

Thanks!

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ pollster = "0.2"
 
 [dev-dependencies]
 pixels-mocks = { path = "internals/pixels-mocks" }
-winit = "0.26"
+winit = "0.27"
 
 [workspace]
 members = [

--- a/examples/conway/Cargo.toml
+++ b/examples/conway/Cargo.toml
@@ -17,5 +17,5 @@ line_drawing = "1.0"
 log = "0.4"
 pixels = { path = "../.." }
 randomize = "3.0"
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"

--- a/examples/custom-shader/Cargo.toml
+++ b/examples/custom-shader/Cargo.toml
@@ -14,5 +14,5 @@ bytemuck = "1.10"
 env_logger = "0.9"
 log = "0.4"
 pixels = { path = "../.." }
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"

--- a/examples/invaders/Cargo.toml
+++ b/examples/invaders/Cargo.toml
@@ -12,12 +12,12 @@ default = ["optimize"]
 [dependencies]
 byteorder = "1.3"
 env_logger = "0.9"
-game-loop = { version = "0.9", features = ["window"] }
+game-loop = { version = "0.10", features = ["winit"] }
 getrandom = "0.2"
 gilrs = "0.9"
 log = "0.4"
 pixels = { path = "../.." }
 randomize = "3.0"
 simple-invaders = { path = "simple-invaders" }
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"

--- a/examples/minimal-egui/Cargo.toml
+++ b/examples/minimal-egui/Cargo.toml
@@ -10,9 +10,9 @@ optimize = ["log/release_max_level_warn"]
 default = ["optimize"]
 
 [dependencies]
-egui = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f" }
-egui-wgpu = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f" }
-egui-winit = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f", default-features = false, features = ["links"] }
+egui = "0.19"
+egui-wgpu = "0.19"
+egui-winit = { version = "0.19", default-features = false, features = ["links"] }
 env_logger = "0.9"
 log = "0.4"
 pixels = { path = "../.." }

--- a/examples/minimal-egui/Cargo.toml
+++ b/examples/minimal-egui/Cargo.toml
@@ -10,11 +10,11 @@ optimize = ["log/release_max_level_warn"]
 default = ["optimize"]
 
 [dependencies]
-egui = { git = "https://github.com/emilk/egui.git", rev = "38a67f86467f16084443258ae5e7761429c00db2" }
-egui-wgpu = { git = "https://github.com/emilk/egui.git", rev = "38a67f86467f16084443258ae5e7761429c00db2" }
-egui-winit = { git = "https://github.com/emilk/egui.git", rev = "38a67f86467f16084443258ae5e7761429c00db2", default-features = false, features = ["links"] }
+egui = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f" }
+egui-wgpu = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f" }
+egui-winit = { git = "https://github.com/emilk/egui.git", rev = "eeeb4b7de2d08b7923a0e19cf1a711b7219afa1f", default-features = false, features = ["links"] }
 env_logger = "0.9"
 log = "0.4"
 pixels = { path = "../.." }
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"

--- a/examples/minimal-web/Cargo.toml
+++ b/examples/minimal-web/Cargo.toml
@@ -14,8 +14,8 @@ default = ["optimize"]
 log = "0.4"
 pixels = { path = "../.." }
 wgpu = "0.13"
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 console_error_panic_hook = "0.1"

--- a/examples/minimal-winit/Cargo.toml
+++ b/examples/minimal-winit/Cargo.toml
@@ -13,5 +13,5 @@ default = ["optimize"]
 env_logger = "0.9"
 log = "0.4"
 pixels = { path = "../.." }
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"

--- a/examples/raqote-winit/Cargo.toml
+++ b/examples/raqote-winit/Cargo.toml
@@ -14,8 +14,8 @@ env_logger = "0.9"
 euclid = "0.22"
 log = "0.4"
 pixels = { path = "../.." }
-winit = "0.26"
-winit_input_helper = "0.12"
+winit = "0.27"
+winit_input_helper = "0.13"
 
 [dependencies.raqote]
 git = "https://github.com/jrmuizel/raqote.git"

